src/components/VisitInput.tsx

```diff
@@ -89,6 +89,5 @@ const VisitInput: React.FC<VisitInputProps> = ({
   );
 };
 
-export default VisitInput;
```

Comment:
Need to update VisitInput import in VisitInput.stories.tsx to import { VisitInput } from "./VisitInput";

Reply:
Good spot! Have updated. Ta.

Reply:
Oh, it's in `VisitInput.test.tsx` too.